src/types.ts

```diff
@@ -11,12 +11,14 @@ export type Template = {
   video?: string;
 };
 
+export type ProjectType = "move" | "dapp";
```

Comment:
Lets use these names
```suggestion
export type ProjectType = "move" | "fullstack";
```